contrib/utxo-tools/utxo_to_sqlite.py

```diff
@@ -111,7 +114,9 @@ def main():
     parser = argparse.ArgumentParser(description=__doc__, formatter_class=argparse.RawDescriptionHelpFormatter)
     parser.add_argument('infile', help='filename of compact-serialized UTXO set (input)')
     parser.add_argument('outfile', help='filename of created SQLite3 database (output)')
-    parser.add_argument('-v', '--verbose', action='store_true', help='show details about each UTXO')
+    parser.add_argument('--verbose', action='store_true', help='show details about each UTXO')
+    parser.add_argument('--spk', choices=['hex', 'raw'], default='hex', help='encode scriptPubKey as hex or raw bytes')
+    parser.add_argument('--txid', choices=['hex', 'raw', 'rawle'], default='hex', help='encode txid as hex, raw bytes (sha256 byteorder), or reversed raw bytes')
```

Comment:
Does `rawle` mean `raw little-endian`? If so, it would be better to be explicit in the description.
```suggestion
    parser.add_argument('--txid', choices=['hex', 'raw', 'rawle'], default='hex', help='encode txid as hex, raw bytes (sha256 byteorder), or reversed raw bytes (litle-endian)')
```

Although `rev_raw` or something similar would also work fine.

Reply:
Agree, done.